docs/guides/create_update_solution.md

```diff
@@ -0,0 +1,339 @@
+---
+ hide:
+  - toc
+---
+!!! abstract "Remember"
+
+    You have to choose the platform and project you want to work. Contact your babylon admin to know more about your options.
+
+    In this example, we will use
+
+    * context_id: `test`
+    * platform_id: `dev`
+
+To deploy a complete solution, you can declare its configuration in yaml files corresponding
+to specific deployment type. Each file contains general information about the deployment:
+
+```yaml
+kind: Organization
+namespace:
+  state_id: "{{state_id}}"
+  context: demo
+  platform:
+    id: dev
+    url: https://dev.api.cosmotech.com/phoenix/v3-0 
+```
+The `kind` key must be one of these: Organization, Solution, Workspace, WebApp, Dataset - it decides which
+resource will be deployed with specification listed below. `namespace` key gives information that identifies
+the deployment: state, context and platform. 
+
+Then, each file declares resource configuration under `spec` key, e.g., in organization deployment file:
+
+```yaml
+kind: Organization
+namespace:
+ state_id: "{{state_id}}"
+ context: demo
+ platform:
+  id: dev
+  url: https://dev.api.cosmotech.com/phoenix/v3-0
+spec:
+ payload:
+  name: My new Organization
+  security: {{security}}
+```
+All keys in this file can be templated with `{{}}` syntax, as `security` section of this file. Corresponding*
+values must be stored in _variables.yaml_ file at the same level that your project:
+
+```yaml
+security:
+  default: viewer
+  accessControlList:
+    - id: user@email.com
+      role: admin
+    - id: user2@email.com
+      role: editor
+```
+Solution deployment file
+```yaml
+kind: Solution
+namespace:
+ state_id: "{{state_id}}"
+ context: demo
+ platform:
+  id: dev
+  url: https://dev.api.cosmotech.com/phoenix/v3-0
+spec:
+ sidecars:
+  azure:
+   run_templates:
+   - id: run_template_id
+     handlers:
+      preRun: true
+ payload:
+  key: "demosolution"
+  name: "My Solution Name"
+  description: "My solution description"
+  repository: brewery_for_continuous
+  version: latest
+  tags:
+  - brewery
+  runTemplates:
+  - id: "run_id"
+    name: "Standard simulation"
+    csmSimulation: AzureWebApp/AzureWebApp_Simulation
+    run: true
+    preRun: true
+  parameterGroups:
+  security:
+   default: none
+   accessControlList:
+   - id: e-mail@cosmotech.com
+     role: admin
+```
+Run templates are enumerated under `sidecars` key which lists every side resources needed for the
+correct functionning of the solution. Run templates scripts must be placed in _run_templates_ folder
+of your project with the following path: _run_templates/<run_template_id>/<handler_id>/<script_file>_
+
+Workspace configuration contains keys needed to deploy a powerBI workspaces, an event hub and an adx database.
```

Comment:
The Workspace deployment file contains keys necessary for deploying a powerBI workspaces, an Event Hub, and an ADX database under the `sidecars` section, specifically in the `azure` section.